update.js

```diff
@@ -141,11 +141,11 @@ async function makeEntry (dir, parsedFileName, oldList) {
   }
 
   if (!build.sha256 || !build.keccak256 || !build.urls || build.urls.length !== 2) {
-    const readFile = util.promisify(fs.readFile)
-    const fileContent = await readFile(absolutePath)
-    build.keccak256 = '0x' + ethUtil.keccak(fileContent).toString('hex')
+    const readfile = promisify(readFile)
+    const fileContent = await readfile(absolutePath)
```

Comment:
Since you are explicitly importing `readFileSync`, you can use [fs.Promises](https://nodejs.org/docs/latest-v16.x/api/fs.html#fspromisesreaddirpath-options) in places where the async version is desirable.

E.g.
```javascript
// assuming the following import:
import { readFile as readFileAsync } from 'node:fs/promises';
// ...
// you can do:
const fileContent = await readFileAsync(absolutePath);
```

```suggestion
    const fileContent = await readFileAsync(absolutePath);
```

Reply:
thank you! good point!